HISTORY.rst

```diff
@@ -1,6 +1,45 @@
 =======
 History
 =======
+3.41.0 (2012-08-04)
+-------------------
+
+Changes
+
+* CCI has a new Metadata ETL task that allows adding fields to existing layouts. Sections can be selected using an index (#2766)
+
+* We added a task to enable an Einstein prediction (thanks, @erikperkins!) (#2778)
+
+* We added standard flows for releasing unlocked packages: release_unlocked_beta and release_unlocked_production (#2768)
+ 
+* We added documentation for using CumulusCI to build managed 2GP packages, 
+  unlocked packages, and extending NPSP and EDA with 2GP packages. (#2768)
+
+* Contributions to CCI now require verification by isort (#2770)
+ 
+* CumulusCI now supports deploying unmanaged dependencies in SFDX source format. (#2735)
+
+* The `create_package_version` task now handles dependencies that use a `zip_url`. (#2735)
+
+* Updates to Github Actions configuration documentation. Thanks @Julian88Tex (#2773)
+
+* CumulusCI now automatically recognizes services and orgs configured via environment variables. See the `docs <https://github.com/SFDO-Tooling/CumulusCI/blob/481ab27446edbfdb038ea62834a6f276dc53bf3d/docs/headless.rst#L9>`_ for more details. (#2676 and #2776)
+
+* We've updated the Push Upgrade tasks (push_list, push_sandbox, etc) task option `start_time` to accept ISO-8601 formatted datetimes. (#2769)
+
+* CumulusCI now automatically recognizes enhanced domains in sandboxes when 
+  establishing a connection via JWT. (#2753)
```

Comment:
I would move this into Issues Closed and rephrase to: "Fixed a couple issues with connecting CumulusCI to sandboxes using enhanced domains." We ended up fixing more than just the JWT issue (see #2765)

We should also include the other change from 2753: You can now specify "sandbox": true on a CUMULUSCI_ORG_* variable in headless environments to indicate that the org you want to connect to is a sandbox. when connecting sandbox orgs in a headless environment

Reply:
Isn't `CUMULUSCI_ORG_` a new feature in this release? If so, this isn't a change: it's just the behaviour of the new feature.

Reply:
No, it's not a new feature. The new feature is that you can use it without explicitly switching to EnvironmentProjectKeychain.